src/TwigComponent/src/ComponentFactory.php

```diff
@@ -71,6 +71,10 @@ public function create(string $name, array $data = []): MountedComponent
 
         // ensure remaining data is scalar
         foreach ($data as $key => $value) {
+            if (\is_object($value) && method_exists($value, '__toString')) {
```

Comment:
```suggestion
            if (\is_object($value) && $value instanceof \Stringable) {
```

Reply:
updated.
also from php docs:
> instanceof does not throw any error if the variable being tested is not an object, it simply returns false. Constants, however, were not allowed prior to PHP 7.3.0.

so removed the is_object :)

Reply:
Perfect! I wondered about that too.